src/main/java/wolox/training/models/Book.java

```diff
@@ -71,6 +73,7 @@ public String getAuthor() {
   }
 
   public void setAuthor(String author) {
+    Preconditions.checkNotNull(author,PreconditionsConstants.NOT_NULL_MESSAGE, "author");
```

Comment:
A simpler solution for this is 

`this.author = Preconditions.checkNotNull(author,PreconditionsConstants.NOT_NULL_MESSAGE, "author");`